scipy/stats/_stats_py.py

```diff
@@ -7617,11 +7628,22 @@ def ks_1samp(x, cdf, args=(), alternative='two-sided', method='auto'):
 
     Returns
     -------
-    statistic : float
-        KS test statistic, either D, D+ or D- (depending on the value
-        of 'alternative')
-    pvalue : float
-        One-tailed or two-tailed p-value.
+    res: KstestResult
+        An object containing attributes:
+
+        statistic : float
+            KS test statistic, either D, D+ or D- (depending on the value
+            of 'alternative')
+        pvalue : float
+            One-tailed or two-tailed p-value.
+        statistic_location: float
+            Value of the stochastic variable at the KS statistic. By
+            definition, the distance between the theoretical and sampled
+            distributions is largest at this location.
```

Comment:
Also, what do you think about "observation" instead of "location"? This is reinforce the idea that this is an element from the sample.